src/main/java/bio/terra/service/filedata/flight/ingest/IngestCopyLoadHistoryStep.java

```diff
@@ -0,0 +1,47 @@
+package bio.terra.service.filedata.flight.ingest;
+
+import bio.terra.service.dataset.Dataset;
+import bio.terra.service.dataset.DatasetService;
+import bio.terra.service.load.LoadService;
+import bio.terra.service.load.LoadService.LoadHistoryIterator;
+import bio.terra.service.load.flight.LoadMapKeys;
+import bio.terra.stairway.*;
+import bio.terra.stairway.exception.DatabaseOperationException;
+import java.time.Instant;
+import java.util.UUID;
+
+public abstract class IngestCopyLoadHistoryStep {
+
+  public IngestCopyLoadHistoryResources getResources(
```

Comment:
Why `public`? It appears to only be used by subclasses.
```suggestion
  protected IngestCopyLoadHistoryResources getResources(
```